exercises/concept/rpn-calculator/tests/rpn-calculator.rs

```diff
@@ -0,0 +1,72 @@
+use rpn_calculator::*;
+use CalculatorInput::*;
+
+#[test]
+fn test_empty_input_returns_none() {
+    let input = vec![];
+    assert_eq!(evaluate(&input), None);
+}
+
+#[test]
+#[ignore]
+fn test_simple_value() {
+    let input = vec![Value(10)];
+    assert_eq!(evaluate(&input), Some(10));
```

Comment:
It'll be much easier to read and write the tests if you write a `fn calculator_input(&str) -> Vec<CalculatorInput>` and use it throughout.

Reply:
Fixed in aaf3df4